python/ray/tune/logger.py

```diff
@@ -122,6 +123,12 @@ def flush(self):
         self._log_syncer.sync_now(force=True)
         self._log_syncer.wait()
 
+    def update_location(self, worker_ip):
+        """Sends the current log directory to the remote node."""
+        if worker_ip != self._log_syncer.worker_ip:
```

Comment:
Will this crash on non-autoscaler clusters?

Also, log a warning if sync is happening?

Reply:
Done; and no, won't crash (1sync_to_worker_if_possible1 catches it)